presentation/src/main/java/com/mashup/gabbangzip/sharedalbum/presentation/ui/common/PicTopBar.kt

```diff
@@ -0,0 +1,232 @@
+    backButtonClicked: () -> Unit,
+    rightIcon1: PicTopBarIcon? = null,
+    rightIcon2: PicTopBarIcon? = null,
+    rightIcon1Clicked: () -> Unit = {},
+    rightIcon2Clicked: () -> Unit = {},
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(vertical = 14.dp, horizontal = 16.dp),
+        contentAlignment = Alignment.Center,
+    ) {
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .wrapContentHeight(),
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            Image(
+                modifier = Modifier
+                    .noRippleClickable { backButtonClicked() }
+                    .size(26.dp),
+                painter = painterResource(id = PicTopBarIcon.BACK.iconRes),
+                contentDescription = stringResource(id = PicTopBarIcon.BACK.desc),
+            )
+            if (titleAlign == PicTopBarTitleAlign.LEFT) {
+                Text(
+                    modifier = Modifier
+                        .padding(start = 6.dp)
+                        .weight(1.0f),
+                    text = titleText,
+                    style = PicTypography.bodyMedium17,
+                    color = Gray80,
+                )
+            }
+            if (rightIcon1 != null) {
+                Image(
+                    modifier = Modifier
+                        .padding(end = 12.dp)
+                        .size(26.dp)
+                        .noRippleClickable { rightIcon1Clicked() },
+                    painter = painterResource(id = rightIcon1.iconRes),
+                    contentDescription = stringResource(id = rightIcon1.desc),
+                )
+            }
+            if (rightIcon2 != null) {
+                Image(
+                    modifier = Modifier
+                        .size(26.dp)
+                        .noRippleClickable { rightIcon2Clicked() },
+                    painter = painterResource(id = rightIcon2.iconRes),
+                    contentDescription = stringResource(id = rightIcon2.desc),
+                )
+            }
+        }
+        if (titleAlign == PicTopBarTitleAlign.CENTER) {
+            Text(
+                modifier = Modifier.fillMaxWidth(1.0f),
+                text = titleText,
+                textAlign = TextAlign.Center,
+                style = PicTypography.bodyMedium16,
+                color = Gray80,
+            )
+        }
+    }
+}
+
+@Composable
+fun PicTextOnlyTopBar(
+    modifier: Modifier = Modifier,
+    titleText: String,
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .wrapContentHeight()
+            .padding(vertical = 14.dp, horizontal = 16.dp),
+        contentAlignment = Alignment.Center,
+    ) {
+        Text(
+            text = titleText,
+            style = PicTypography.bodyMedium16,
+            color = Gray80,
+        )
+    }
+}
```

Comment:
```suggestion
@Composable
fun PicTextOnlyTopBar(
    modifier: Modifier = Modifier,
    titleText: String,
) {
    Text(
        modifier = modifier
            .fillMaxWidth()
            .wrapContentHeight()
            .padding(vertical = 14.dp, horizontal = 16.dp),
        text = titleText,
        style = PicTypography.bodyMedium16,
        color = Gray80,
        textAlign = TextAlign.Center,
    )
}
```
여기두 Box 로 감싸지 않고 개발할 수 있을 것 같은데 Composable 함수를 Box 가 필요한 이유가 없다면 최적화해보는 것은 어떨까?

Reply:
굳.22 5649eac43c7f73dde161635e986909f69beeef8a